charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -0,0 +1,114 @@
+        - --cloud-config=/srv/cloudprovider/kubeconfig
+        - --cluster-cidr={{ .Values.podNetwork }}
+        - --cluster-name={{ .Values.clusterName }}
+        - --concurrent-service-syncs=1
+        - --configure-cloud-routes=true
+        {{- include "cloud-controller-manager.featureGates" . | trimSuffix "," | indent 8 }}
+        - --kubeconfig=/var/lib/cloud-controller-manager/kubeconfig
+        - --leader-elect=true
+        {{- if semverCompare ">= 1.13" .Values.kubernetesVersion }}
+        - --secure-port={{ include "cloud-controller-manager.port" . }}
+        - --port=0
+        {{- end }}
+        {{- if semverCompare ">= 1.12" .Values.kubernetesVersion }}
+        - --authentication-kubeconfig=/var/lib/cloud-controller-manager/kubeconfig
+        - --authorization-kubeconfig=/var/lib/cloud-controller-manager/kubeconfig
+        - --tls-cert-file=/var/lib/cloud-controller-manager-server/cloud-controller-manager-server.crt
+        - --tls-private-key-file=/var/lib/cloud-controller-manager-server/cloud-controller-manager-server.key
+        {{- end }}
+        - --tls-cipher-suites={{ include "kubernetes.tlsCipherSuites" . | replace "\n" "," | trimPrefix "," }}
+        - --use-service-account-credentials
+        - --v=2
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            {{- if semverCompare ">= 1.13" .Values.kubernetesVersion }}
+            scheme: HTTPS
+            {{- else }}
+            scheme: HTTP
+            {{- end }}
+            port: {{ include "cloud-controller-manager.port" . }}
+          successThreshold: 1
+          failureThreshold: 2
+          initialDelaySeconds: 15
+          periodSeconds: 10
+          timeoutSeconds: 15
+        ports:
+        - containerPort: {{ include "cloud-controller-manager.port" . }}
+          name: metrics
+          protocol: TCP
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        {{- if .Values.resources }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+        {{- end }}
+        volumeMounts:
+        - name: cloud-controller-manager
+          mountPath: /var/lib/cloud-controller-manager
+        - name: cloud-controller-manager-server
+          mountPath: /var/lib/cloud-controller-manager-server
+        - name: cloudprovider
+          mountPath: /srv/cloudprovider        
+        - name: etc-ssl
+          mountPath: /etc/ssl
+          readOnly: true
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - name: cloud-controller-manager
+        secret:
+          secretName: cloud-controller-manager
+      - name: cloud-controller-manager-server
+        secret:
+          secretName: cloud-controller-manager-server
+      - name: cloudprovider
+        secret:
+          secretName: cloudprovider          
+      - name: etc-ssl
+        hostPath:
```

Comment:
use emptyDir, hostPath needs privilege escalation on OCP

Reply:
This is not needed anyway and will be removed.